### ISSUE TITLE
sqlite3 does not work, unless manually installed

### ISSUE THREAD

**Issue by robobun** [oven-sh/bun#5187]
Can't get bun to work with sequelize. I have added the sqlite3 package with `bun add sqlite3` but it doesn't seem to see it.

![image.png](https://cdn.discordapp.com/attachments/1150888134918295682/1150888135048306839/image.png)

> After some tinkering, I've figured it out, basically I had to manually build sqlite3 manually using bun install --target_platform=linux --target_libc=glibc from the sqlite3 directory in my node_modules.

_Originally reported on Discord: [`bun with ORM (sequelize)`](https://discord.com/channels/876711213126520882/1150888134918295682/1150888134918295682)_

**Comment by ismail424** [comment]
Any fixes yeet?

**Comment by SeqSEE** [comment]
> Any fixes yeet?

Need to add "sqlite3" to trustedDependencies before installing for the gyp postinstall step to run: https://bun.sh/docs/cli/install#trusted-dependencies

There is a default allow-list being added which will include sqlite3:  https://github.com/oven-sh/bun/issues/4959

**Comment by ismail424** [comment]
You are right,

I solved it by

`cd node_modules/sqlite3`
`bun install`

Done.

**Comment by SeqSEE** [comment]
> You are right,
> 
> I solved it by
> 
> `cd node_modules/sqlite3` `bun install`
> 
> Done.

After  you add it to `trustedDependencies` you can just do  `bun remove sqlite3` and then `bun add sqlite3`.

**Comment by cgslivre** [comment]
> > You are right,
> > I solved it by
> > `cd node_modules/sqlite3` `bun install`
> > Done.
> 
> After you add it to `trustedDependencies` you can just do `bun remove sqlite3` and then `bun add sqlite3`.

Fail on macos!

package.json:
```json
{
  "name": "bot-bun",
  "module": "index.ts",
  "type": "module",
  "devDependencies": {
    "bun-types": "latest"
  },
  "peerDependencies": {
    "typescript": "^5.0.0"
  },
  "dependencies": {
    "knex": "^3.0.1"
  },
  "trustedDependencies": [
    "sqlite3"
  ]
}
```

index.ts
```typescript
import knex from "knex";

const db = knex({
    client: "sqlite3",
    connection: {
        filename: "./app.sqlite3",
    },
    useNullAsDefault: true,
});

console.log(await db.table("documents").first());
```

Steps:
```sh
bun remove sqlite3
bun add sqlite3
```

Result:
![print-202310141012](https://github.com/oven-sh/bun/assets/2656420/a01a215a-ee0c-45df-857c-c07e766c4682)

But, if run "bun install"...

```sh
bun add sqlite3 && cd node_modules/sqlite3 && bun install && cd ../../ 
```
Result:
![print-202310141013](https://github.com/oven-sh/bun/assets/2656420/48ac653a-0994-44fe-a23e-05cbbfb30215)

**Comment by Electroid** [comment]
Duplicate of #4959 (we will fix this)

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway and @paperdave in #7132

This will be part of the Bun v1.0.17 release, which ships in a few hours from the time of writing.
